.storybook/main.js

```diff
@@ -108,3 +110,25 @@ function overrideDefaultBabelLoader(rules) {
 
   loader.options.customize = customLoaderPath;
 }
+
+/**
+ * @returns {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}
+ */
+function getCodesandboxBabelOptions() {
+  const allPackageInfo = getAllPackageInfo();
+
+  /** @type {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}  */
+  const initialValue = {};
+  return Object.values(allPackageInfo).reduce((acc, cur) => {
+    if (isConvergedPackage(cur.packageJson)) {
```

Comment:
is this check needed ? this storybook is used only for v9

Reply:
it's to avoid having non v9 dependencies in the configuration... it's not that important but I would prefer not to have @fluentui/react in the babel plugin config of a v9 storybook addon :)